.github/workflows/pr-merged.yaml

```diff
@@ -0,0 +1,26 @@
+name: build
+on:
+  pull_request_target:
+    types: [closed]
+    branches: 
+    - 'main'
+    paths-ignore:
+      - '**.md' ### Ignore running when README.MD changed.
+      - '.github/workflows/*' ### Ignore running when files under path: .github/workflows/* changed.
+      
+jobs:
+  pr-merged:
+    if: ${{ github.event.pull_request.merged == true }} ## Skip if not merged 
+    uses: kubescape/workflows/.github/workflows/incluster-comp-pr-merged.yaml@main
```

Comment:
how did you test this? from my tests you cannot really use the "secrets: inherit" from libs in a different org (kubescape VS yrs147)

Reply:
Oh, sorry. This part is not tested. I'll comment it for now.